package.json

```diff
@@ -77,6 +77,6 @@
   },
   "lint-staged": {
     "*.js": "eslint --cache --fix",
-    "*.{js,jsx,css,scss,md,rb,erb,haml}": "prettier --write"
```

Comment:
why no erb here? did it not work as well as I thought?

Reply:
Prettier just throws an error if you try to run it against an erb file. There might need to be some other configuration? But as far as I could tell, ERB straight up isn't supported at all.

Reply:
Some internet research indicates that Prettier does not yet have meaningful support for ERB: https://github.com/prettier/plugin-ruby/issues/371. There's a plugin linked at the end of that thread, but the plugin has only one commit, and that was just a few weeks ago.